sdk/cognitiveservices/arm-cognitiveservices/package.json

```diff
@@ -4,9 +4,9 @@
   "description": "CognitiveServicesManagementClient Library with typescript type definitions for node.js and browser.",
   "version": "3.1.0",
```

Comment:
Looks like the version number hasn't been bumped.  If you're planning to ship an update of this, I'd say the new version should be 4.0.0 because of the new ms-rest-js 2.0.